hr_timesheet_report_not_valued/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Report Not Valued",
+    "summary": """
+        Adds a report that hides the time field.
+    """,
+    "version": "12.0.1.0.0",
```

Comment:
Wrong version, please fix it. You should also re-generate README info

Reply:
Changed and updated manifest version

Reply:
Changed and updated <report> tag. Also updated README